ttnn/cpp/ttnn/operations/data_movement/non_zero_indices/non_zero_indices.hpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <vector>
+
+#include "ttnn/run_operation.hpp"
```

Comment:
Please remove this header, it is not needed and is heavy

Reply:
removed. Thanks